test/unit/math/rev/mat/prob/normal_id_glm_lpdf_test.cpp

```diff
@@ -115,6 +115,79 @@ TEST(ProbDistributionsNormalIdGLM, glm_matches_normal_id_vars) {
   }
 }
 
+TEST(ProbDistributionsNormalIdGLM, broadcast_x) {
```

Comment:
It would be nice to test these with the distribution test framework.  

If that's not possible, we need tests at the very least for forward-mode autodiff.  It should just work if the `operands_and_partials` code is written generally enough.  So it's also a nice test for that.

The autodiff testing framework can only currently handle up to three-argument functions, so that's not an option.  It would require 2^4 or so definitions for 4-arg and 2^5 for 5-arg, but it's probably worth it to test things like this.  But I'm *not* asking you to do this for this PR!

Reply:
I never worked with forward, so I am probably not the best person for this. But I can try. Are you suggesting that forward mode tests should be added here or in a separate PR?

Reply:
They should've been added here before this was ever accepted the first time.  But now that everything other than solvers are working for forward mode, I'd like to keep everything else in synch.  

I'd be OK adding them in a separate test.  Would you mind creating an issue listing the GLMs that need forward-mode tests and assign it to me?  Thanks.

Reply:
The issue is here: https://github.com/stan-dev/math/issues/1501